src/content/en/updates/2019/08/get-started-with-gpu-compute-on-the-web.md

```diff
@@ -388,7 +388,8 @@ const shaderModule = device.createShaderModule({
     [[group(0), binding(1)]] var<storage, read> secondMatrix : Matrix;
     [[group(0), binding(2)]] var<storage, write> resultMatrix : Matrix;
 
-    [[stage(compute)]] fn main([[builtin(global_invocation_id)]] global_id : vec3<u32>) {
+    [[stage(compute), workgroup_size(64)]]
```

Comment:
The dispatch size gets implicitly multiplied by this number. I suggest doing `workgroup_size(8, 8)` and dividing the dispatch size by 8 in both dimensions.

We should have quite a massive speedup on the shader with this (in the order of 10 to 60 depending on your GPU) so it might be worth making the graphs again.

Reply:
Done.